### ISSUE TITLE
Impaired ability to organize

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5853]
**Preferred term label:**
Impaired ability to organize

**Synonyms**
Impaired organizational skills

**Definition (free text, please give PubMed ID)**
Impaired ability to o bring order to information, actions, or materials to achieve a goal or to follow an established organized routine.
Source: NCIT:C121469

Parent term (use hpo.jax.org/app)
HP_0030216 Inertia ?
Def is: Reduction of goal-directed behaviors linked to the impairment in frontal executive functions (planning of an action for example).

Related to #5852 and #5850 
Maybe these terms should all be in the same branch. Maybe this term should be a child of Executive dysfunction?

This is needed for the Kids First INCLUDE supplement.
